docs/troubleshooting_v1.md

```diff
@@ -2,10 +2,14 @@
 
 This error returned [here](https://github.com/Layr-Labs/eigenda-proxy/blob/97b71bda6e2136a6c0d71be2f7e9642dc840fefc/verify/cert.go#L125) happens when verifying a v1 cert, where the batch hash in the cert does not match the hash that was computed when the cert was bridged onchain by the EigenDA disperser.
 
+> Note: This error is only relevant for op stacks. Nitro does not suffer from this error because the sequencer inbox is an actual contract which ensures that the cert is valid, as opposed to op's batcher inbox which is just an EOA and hence cannot ensure atomicity of the confirmBatch transaction on the EigenDAServiceManager and the cert being posted to the inbox.
```

Comment:
```suggestion
> Note: This error is only relevant for op stacks. Nitro layer2s that settle to an Ethereum network do not suffer from this error because the sequencer inbox is an actual contract which verifies the cert against EigenDA service manager state, as opposed to op's batcher inbox which is just an EOA. Arbitrum Layer 3s that use EigenDA are also susceptible to the mismatch error since certificates can't be verified on the Layer 2 sequencer inbox. 
```

Reply:
[8b82359](https://github.com/Layr-Labs/eigenda-proxy/pull/322/commits/8b823591c09b7d0a51fb3880d9fbdc28f05728d2)